### PR TITLE
Corrected various typos in contributing docs.

### PR DESCRIPTION
**Pull request by arslannoorpansota**

### CODE REVIEW

File: docs/internals/contributing/committing-code.txt [hunk 1]
```diff
@@ -32,8 +32,8 @@ alias will be helpful::
     [alias]
         pr = !sh -c \"git fetch upstream pull/${1}/head:pr/${1} && git checkout pr/${1}\"
 
-Add it to your ``~/.gitconfig``, and set ``upstream`` to be ``django/django``.
-Then you can run ``git pr ####`` to checkout the corresponding pull request.
+Add it to your ``~/.gitconfig``, and set ``upstream`` it to be ``django/django``.
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/committing-code.txt:35]: Why this change?

File: docs/internals/contributing/bugs-and-features.txt [hunk 2]
```diff
@@ -125,7 +125,7 @@ How we make decisions
 =====================
 
 Whenever possible, we strive for a rough consensus. To that end, we'll often
-have informal votes on |django-developers| about a feature. In these votes we
+have informal votes on |django-developers| about a feature. In these votes, we
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/bugs-and-features.txt:128]: This introducing clause is just three words--it's fine without a comma.

File: docs/internals/contributing/committing-code.txt [hunk 3]
```diff
@@ -59,7 +59,7 @@ Once you're ready:
     $ # Delete the pull request branch.
     $ git branch -d pr/xxxx
 
-Force push to the branch after rebasing on main but before merging and pushing
+Force push to the branch after rebasing on the main but before merging and pushing
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/committing-code.txt:62]: I think this edit is unnecessary.

File: docs/internals/contributing/committing-code.txt [hunk 4]
```diff
@@ -211,13 +211,13 @@ Nobody's perfect; mistakes will be committed.
 But try very hard to ensure that mistakes don't happen. Just because we have a
 reversion policy doesn't relax your responsibility to aim for the highest
 quality possible. Really: double-check your work, or have it checked by
-another committer, **before** you commit it in the first place!
+another committer **before** you commit it in the first place!
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/committing-code.txt:214]: I would revert.

File: docs/internals/contributing/committing-code.txt [hunk 5]
```diff
@@ -211,13 +211,13 @@ Nobody's perfect; mistakes will be committed.
...
 
 When a mistaken commit is discovered, please follow these guidelines:
 
 * If possible, have the original author revert their own commit.
 
-* Don't revert another author's changes without permission from the
+* Don't revert to another author's changes without permission from the
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/committing-code.txt:220]: I would revert.

File: docs/internals/contributing/index.txt [hunk 6]
```diff
@@ -65,7 +65,7 @@ Django community and others to maintain a great ecosystem to work in:
   friendly and helpful atmosphere. If you're new to the Django community,
   you should read the `posting guidelines`_.
 
-* Join the `#django IRC channel`_ on Libera.Chat and answer questions. By
+* Join the `#django IRC channel`_ on Libera. Chat and answer questions. By
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/index.txt:68]: Libera.chat is a web address and brand name, so I would revert.

File: docs/internals/contributing/localizing.txt [hunk 7]
```diff
@@ -14,9 +14,9 @@ Translations
 Translations are contributed by Django users worldwide. The translation work is
 coordinated at `Transifex`_.
 
-If you find an incorrect translation or want to discuss specific translations,
+If you find any incorrect translation or want to discuss specific translations,
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/localizing.txt:17]: I would revert.

File: docs/internals/contributing/new-contributors.txt [hunk 8]
```diff
@@ -113,7 +113,7 @@ some advice to make your work on Django more useful and rewarding.
   Sometimes it can be scary to put your opinion out to the world and say "this
   ticket is correct" or "this patch needs work", but it's the only way the
   project moves forward. The contributions of the broad Django community
-  ultimately have a much greater impact than that of any one person. We can't
+  ultimately have a much greater impact than that of anyone person. We can't
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/new-contributors.txt:116]: The proposed edit is incorrect.

File: docs/internals/contributing/triaging-tickets.txt [hunk 9]
```diff
@@ -2,16 +2,16 @@
 Triaging tickets
 ================
 
-Django uses Trac_ for managing the work on the code base. Trac is a
+Django uses Trac_ for managing the work on the codebase. Trac is a
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:5]: I would revert; not sure that 'codebase' is a word.

File: docs/internals/contributing/triaging-tickets.txt [hunk 10]
```diff
@@ -2,16 +2,16 @@
...
 would like to see added. As in any garden, sometimes there are weeds to be
 pulled and sometimes there are flowers and vegetables that need picking. We need
-your help to sort out one from the other, and in the end we all benefit
+your help to sort out one from the other, and in the end, we all benefit
 together.
 
-Like all gardens, we can aspire to perfection but in reality there's no such
-thing. Even in the most pristine garden there are still snails and insects.
-In a community garden there are also helpful people who -- with the best of
+Like all gardens, we can aspire to perfection but in reality, there's no such
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:12]: Comma before the coordinating "but" is more important!
```suggestion
Like all gardens, we can aspire to perfection, but in reality, there's no such
```

File: docs/internals/contributing/triaging-tickets.txt [hunk 11]
```diff
@@ -2,16 +2,16 @@
...
+thing. Even in the most pristine garden, there are still snails and insects.
+In a community garden, there are also helping people who -- with the best of
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:14]: Why this edit?

File: docs/internals/contributing/triaging-tickets.txt [hunk 12]
```diff
@@ -41,8 +41,8 @@ tickets have patches, but those patches don't meet all the requirements of a
 One way to help out is to *triage* tickets that have been created by other
 users.
 
-Most of the workflow is based around the concept of a ticket's
-:ref:`triage stages <triage-stages>`. Each stage describes where in its
+Most of the workflow is based on the concept of a ticket's
+:ref:`triage stages <triage-stages>`. Each stage describes wherein its
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:45]: The proposed edit is incorrect.

File: docs/internals/contributing/triaging-tickets.txt [hunk 13]
```diff
@@ -41,8 +41,8 @@ tickets have patches, but those patches don't meet all the requirements of a
 One way to help out is to *triage* tickets that have been created by other
 users.
 
-Most of the workflow is based around the concept of a ticket's
-:ref:`triage stages <triage-stages>`. Each stage describes where in its
+Most of the workflow is based on the concept of a ticket's
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:44]: I would revert.

File: docs/internals/contributing/triaging-tickets.txt [hunk 14]
```diff
@@ -58,7 +58,7 @@ We've got two roles in this diagram:
 * Committers: people with commit access who are responsible for making the
   final decision to merge a patch.
 
-* Ticket triagers: anyone in the Django community who chooses to
+* Ticket triage: anyone in the Django community who chooses to
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:61]: This is now less analogous with its surroundings; I would revert.

File: docs/internals/contributing/triaging-tickets.txt [hunk 15]
```diff
@@ -82,7 +82,7 @@ By way of example, here we see the lifecycle of an average ticket:
 * Alice updates the pull request, fixing the implementation. She removes the
   "patch needs improvement" flag.
 
-* Daisy reviews the pull request and marks the ticket as "Ready for checkin".
+* Daisy reviews the pull request and marks the ticket as "Ready for check-in".
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:85]: I'm not certain if it's worth changing these if it's going to become out of sync with Trac, where it's called "checkin".

File: docs/internals/contributing/triaging-tickets.txt [hunk 16]
```diff
@@ -117,8 +117,8 @@ Beyond that there are several considerations:
   means you could safely start writing a patch for it. This is generally more
   true for the case of accepted bugs than accepted features. A ticket for a bug
   that has been accepted means that the issue has been verified by at least one
-  triager as a legitimate bug - and should probably be fixed if possible. An
-  accepted new feature may only mean that one triager thought the feature would
+  trigger as a legitimate bug - and should probably be fixed if possible. An
+  accepted new feature may only mean that one trigger thought the feature would
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:121]: Edits are incorrect.

File: docs/internals/contributing/triaging-tickets.txt [hunk 17]
```diff
@@ -132,7 +132,7 @@ Beyond that there are several considerations:
 
 * **Accepted + Has Patch + Needs ...**
 
-  This means the ticket has been reviewed, and has been found to need further
+  This means the ticket has been reviewed and has been found to need further
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:135]: "has been found ..." is not an independent clause, so the edit is incorrect.

File: docs/internals/contributing/triaging-tickets.txt [hunk 18]
```diff
@@ -192,7 +192,7 @@ Patch needs improvement
 -----------------------
 
 This flag means that although the ticket *has* a patch, it's not quite
-ready for checkin. This could mean the patch no longer applies
+ready for check-in. This could mean the patch no longer applies
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:195]: as above

File: docs/internals/contributing/triaging-tickets.txt [hunk 19]
```diff
@@ -226,7 +226,7 @@ easier to find.
 Severity
 --------
 
-The *severity* attribute is used to identify blockers, that is, issues which
+The *severity* attribute is used to identify blockers, that is, issues that
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:229]: 👍🏻

File: docs/internals/contributing/triaging-tickets.txt [hunk 20]
```diff
@@ -254,8 +254,8 @@ new contributions are made to the ticket.
 Keywords
 --------
 
-With this field you may label a ticket with multiple keywords. This can be
-useful, for example, to group several tickets of a same theme. Keywords can
+With this field, you may label a ticket with multiple keywords. This can be
+useful, for example, to group several tickets of the same theme. A Keywords can
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:258]: "A Keywords" is not correct.

File: docs/internals/contributing/triaging-tickets.txt [hunk 21]
```diff
@@ -305,7 +305,7 @@ A ticket can be resolved in a number of ways:
       submit support queries as tickets).
 
 * wontfix
-      Used when a someone decides that the request isn't appropriate for
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:308]: 👍🏻

File: docs/internals/contributing/triaging-tickets.txt [hunk 22]
```diff
@@ -328,7 +328,7 @@ A ticket can be resolved in a number of ways:
       should be reopened when more information is supplied.
 
 If you believe that the ticket was closed in error -- because you're
-still having the issue, or it's popped up somewhere else, or the triagers have
+still having the issue, or it's popped up somewhere else, or the triggers have
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:331]: Triagers is correct; it refers to the people who triage tickets.

File: docs/internals/contributing/triaging-tickets.txt [hunk 23]
```diff
@@ -414,8 +414,7 @@ Bisecting a regression
 
 .. highlight:: console
 
-A regression is a bug that's present in some newer version of Django but not in
-an older one. An extremely helpful piece of information is the commit that
+Regression is a bug that's present in some newer versions of Django but not in older ones. An extremely helpful piece of information is the commit that
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:417]: Why this edit? Please revert.

File: docs/internals/contributing/triaging-tickets.txt [hunk 24]
```diff
@@ -436,9 +435,9 @@ Next, we mark the current point in history as being "bad" since the test fails::
 
 Now, we need to find a point in git history before the regression was
 introduced (i.e. a point where the test passes). Use something like
-``git checkout HEAD~100`` to checkout an earlier revision (100 commits earlier,
+``git checkout HEAD~100`` to check out an earlier revision (100 commits earlier,
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:438]: 👍🏻

File: docs/internals/contributing/writing-documentation.txt [hunk 25]
```diff
@@ -2,14 +2,14 @@
...
-We place a high importance on consistency and readability of documentation.
+We place high importance on the consistency and readability of documentation.
 After all, Django was created in a journalism environment! So we treat our
 documentation like we treat our code: we aim to improve it as often as
 possible.
 
 Documentation changes generally come in two forms:
 
-* General improvements: typo corrections, error fixes and better
+* General improvements: typo corrections, error fixes, and better
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/writing-documentation.txt:12]: This reads alright without a serial comma, so I would revert.

File: docs/internals/contributing/writing-documentation.txt [hunk 26]
```diff
@@ -29,7 +29,7 @@ Django release.
 If you'd like to start contributing to our docs, get the development version of
 Django from the source code repository
 (see :ref:`installing-development-version`). The development version has the
-latest-and-greatest documentation, just as it has latest-and-greatest code.
+latest-and-greatest documentation, just as it has the latest-and-greatest code.
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/writing-documentation.txt:32]: 🚀

File: docs/internals/contributing/writing-documentation.txt [hunk 27]
```diff
@@ -386,7 +386,7 @@ something like this::
 
 Put the changed annotation notes at the bottom of a section, not the top.
 
-Also, avoid referring to a specific version of Django outside a
+Also, avoid referring to a specific version of Django outside
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/writing-documentation.txt:389]: Why this edit?

File: docs/internals/contributing/writing-documentation.txt [hunk 28]
```diff
@@ -416,7 +416,7 @@ AdvanceCOMP's ``advpng``:
    $ advpng -z4 `find . -type f -not -path "./_build/*" -name "*.png"`
 
 This is based on OptiPNG version 0.7.5. Older versions may complain about the
-``-strip all`` option being lossy.
+``-strip all`` an option being lossy.
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/writing-documentation.txt:419]: Why this edit?

File: docs/internals/contributing/bugs-and-features.txt [hunk 29]
```diff
@@ -142,9 +142,9 @@ Although these votes on |django-developers| are informal, they'll be taken very
 seriously. After a suitable voting period, if an obvious consensus arises we'll
 follow the votes.
 
-However, consensus is not always possible. If consensus cannot be reached, or
+However, the consensus is not always possible. If consensus cannot be reached, or
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/bugs-and-features.txt:145]: Why this edit?

File: docs/internals/contributing/localizing.txt [hunk 30]
```diff
@@ -55,7 +55,7 @@ Formats
 =======
 
 You can also review ``conf/locale/<locale>/formats.py``. This file describes
-the date, time and numbers formatting particularities of your locale. See
+the date, time, and numbers formatting particularities of your locale. See
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/localizing.txt:58]: This reads alright without a serial comma. Smaller diffs are more mergeable.

**Review by jacobtylerwalls — COMMENTED** [review]
Thanks for your interest in sprucing up these docs. You identified a handful of problems worth fixing. I left some comments identifying things I don't think are quite right. All the best, Jacob

File: docs/internals/contributing/bugs-and-features.txt [hunk 31]
```diff
@@ -142,9 +142,9 @@ Although these votes on |django-developers| are informal, they'll be taken very
...
 if the discussion toward a consensus fizzles out without a concrete decision,
-the decision may be deferred to the :ref:`technical board <technical-board>`.
+the decision may be referred to the :ref:`technical board <technical-board>`.
```
Comment (felixxm) [inline docs/internals/contributing/bugs-and-features.txt:147]: That's not a typo.

File: docs/internals/contributing/committing-code.txt [hunk 32]
```diff
@@ -32,8 +32,8 @@ alias will be helpful::
     [alias]
         pr = !sh -c \"git fetch upstream pull/${1}/head:pr/${1} && git checkout pr/${1}\"
 
-Add it to your ``~/.gitconfig``, and set ``upstream`` to be ``django/django``.
-Then you can run ``git pr ####`` to checkout the corresponding pull request.
+Add it to your ``~/.gitconfig``, and set ``upstream`` it to be ``django/django``.
+Then you can run ``git pr ####`` to check out the corresponding pull request.
```
Comment (felixxm) [inline docs/internals/contributing/committing-code.txt:36]: That's not a typo.

File: docs/internals/contributing/index.txt [hunk 33]
```diff
@@ -4,7 +4,7 @@ Contributing to Django
 
 Django is a community that lives on its volunteers. As it keeps growing, we
 always need more people to help others. You can contribute in many ways, either
-on the framework itself or in the wider ecosystem.
+to the framework itself or in the wider ecosystem.
```
Comment (felixxm) [inline docs/internals/contributing/index.txt:7]: This is also correct.

**Comment by felixxm** [comment]
@arslannoorpansota Thanks for this patch :+1: However most of your edits are not typos. I also think that the current punctuation is correct.

File: docs/internals/contributing/triaging-tickets.txt (hunk 11 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:14]: Either it should be "helpful person" or helping people!

File: docs/internals/contributing/triaging-tickets.txt (hunk 9 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/triaging-tickets.txt:5]: Depends on the source ;) 

Here's a mainstream English dictionary:
Screen Shot 2021-12-14 at 4 09 52 PM

File: docs/internals/contributing/triaging-tickets.txt (hunk 11 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/triaging-tickets.txt:14]: Why can't folks be helpful? :D

File: docs/internals/contributing/triaging-tickets.txt (hunk 9 above)
Comment (smithdc1) [reply docs/internals/contributing/triaging-tickets.txt:5]: Code base / codebase is not in the style guide. Source code is though!

File: docs/internals/contributing/committing-code.txt (hunk 1 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/committing-code.txt:35]: reverting

File: docs/internals/contributing/committing-code.txt (hunk 32 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/committing-code.txt:36]: Sorry, I got it the wrong way. Reverting it

File: docs/internals/contributing/new-contributors.txt (hunk 8 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/new-contributors.txt:116]: Sorry, I got it the wrong way. Reverting...

File: docs/internals/contributing/triaging-tickets.txt (hunk 9 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:5]: changing it to source code then :)

File: docs/internals/contributing/triaging-tickets.txt (hunk 11 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:14]: "helping people" resonates more I guess. :)
but I would revert if you wouldn't agree!

File: docs/internals/contributing/triaging-tickets.txt (hunk 23 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:417]: I think it makes sense. There are multiple older versions :D

File: docs/internals/contributing/triaging-tickets.txt (hunk 13 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:44]: Lemme revert :p

File: docs/internals/contributing/localizing.txt (hunk 30 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/localizing.txt:58]: Reverting...

File: docs/internals/contributing/bugs-and-features.txt (hunk 31 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/bugs-and-features.txt:147]: Reverting... :D

File: docs/internals/contributing/index.txt (hunk 33 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/index.txt:7]: Doesn't "contribute to" sounds more accurate than "contribute on"?

File: docs/internals/contributing/triaging-tickets.txt (hunk 23 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/triaging-tickets.txt:417]: Oh, I didn't even notice that part. I was looking at "Regression" instead of "A regression". "Regression is a bug" is not quite right.

Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:417]: Changing "Regression" to "A regression" :p

File: docs/internals/contributing/triaging-tickets.txt [hunk 34]
```diff
@@ -414,8 +414,7 @@ Bisecting a regression
 
 .. highlight:: console
 
-A regression is a bug that's present in some newer version of Django but not in
-an older one. An extremely helpful piece of information is the commit that
+A Regression is a bug that's present in some newer versions of Django but not in older ones. An extremely helpful piece of information is the commit that
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:417]: I wouldn't capitalize Regression, and you should wrap the line at 79 chars.

File: docs/internals/contributing/writing-documentation.txt [hunk 35]
```diff
@@ -351,15 +351,15 @@ Documenting new features
...
+    clearly designates the features that are only available in the Django
     development version. Assume documentation readers are using the latest
     release, not the development version.
 
 Our preferred way for marking new features is by prefacing the features'
 documentation with: "``.. versionadded:: X.Y``", followed by a mandatory
 blank line and an optional description (indented).
 
-General improvements, or other changes to the APIs that should be emphasized
+General improvements or other changes to the APIs that should be emphasized
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/writing-documentation.txt:362]: Can you explain this change? We're not necessarily glossing "general improvements," and if we were, there would need to be another comma on the other side.

File: docs/internals/contributing/triaging-tickets.txt (hunk 11 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/triaging-tickets.txt:14]: I guess I don't see a strong reason for the change. If anything it makes this garden analogy a bit worse, because the whole point is that they _aren't_ helping the garden, despite having a helpful disposition. It's about being patient with folks who actually aren't helping sometimes.

**Review by jacobtylerwalls — COMMENTED** [review]
Thanks for the updates.

File: docs/internals/contributing/index.txt (hunk 33 above)
Comment (felixxm) [reply docs/internals/contributing/index.txt:7]: "on" is correct IMO.

File: docs/internals/contributing/triaging-tickets.txt (hunk 11 above)
Comment (felixxm) [reply docs/internals/contributing/triaging-tickets.txt:14]: Also, we try to avoid trivial wording changes unless it's a big problem as it creates extra work for documentation translators.

File: .gitignore [hunk 36]
```diff
@@ -16,3 +16,4 @@ tests/coverage_html/
 tests/.coverage
 build/
 tests/report/
+.idea/
```
Comment (felixxm) [inline .gitignore:19]: Please revert this.

File: docs/internals/contributing/writing-documentation.txt [hunk 37]
```diff
@@ -207,7 +207,7 @@ documentation:
   "last bit" of that path. So ``:mod:`~django.contrib.auth``` will
   display a link with the title "auth".
 
-* Use :mod:`~sphinx.ext.intersphinx` to reference Python's and Sphinx'
+* Use :mod:`~sphinx.ext.intersphinx` to reference Python's and Sphinx's
```
Comment (felixxm) [inline docs/internals/contributing/writing-documentation.txt:210]: IMO `s` is not missing here. I would revert.

File: docs/internals/contributing/index.txt [hunk 38]
```diff
@@ -26,7 +26,7 @@ The work on Django itself falls into three major areas:
 
 **Localizing Django** 🗺️
     Django is translated into over 100 languages - There's even some
-    translation for Klingon?! The i18n team are always looking for translators
+    translation for Klingon?! The i18n team is always looking for translators
```
[the remark this answers is not in the document]
Comment (jacobtylerwalls) [reply docs/internals/contributing/index.txt:29]: I pretty much exclusively treat groups as singular. What do you think?
https://style.mla.org/verbs-with-collective-nouns/

[the remark this answers is not in the document]
Comment (smithdc1) [reply docs/internals/contributing/index.txt:29]: Not sure.... 

https://editorsmanual.com/articles/collective-nouns-singular-or-plural/

This likely explains why we have a different view. 

We're aiming for US English so let's go with your suggestion. 👍

[the remark this answers is not in the document]
Comment (jacobtylerwalls) [reply docs/internals/contributing/index.txt:29]: thanks for the link--I learned something! :D

File: docs/internals/contributing/index.txt (hunk 33 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/index.txt:7]: ok. I will revert!

File: docs/internals/contributing/writing-documentation.txt (hunk 35 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/writing-documentation.txt:362]: I don't like to use a comma with "or" very often. BTW I would revert it if you say!

File: docs/internals/contributing/committing-code.txt [hunk 39]
```diff
@@ -16,7 +16,7 @@ requests.
 
 When committing a pull request, make sure each individual commit matches the
 commit guidelines described below. Contributors are expected to provide the
-best pull requests possible. In practice however, committers - who will likely
+best pull requests possible. In practice, however, committers - who will likely
```
[the remark this answers is not in the document]
Comment (arslannoorpansota) [reply docs/internals/contributing/committing-code.txt:19]: Done!

File: docs/internals/contributing/triaging-tickets.txt [hunk 40]
```diff
@@ -315,7 +315,7 @@ A ticket can be resolved in a number of ways:
 
 * duplicate
       Used when another ticket covers the same issue. By closing duplicate
-      tickets, we keep all the discussion in one place, which helps
+      tickets, we keep all the discussions in one place, which helps
```
[the remark this answers is not in the document]
Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:318]: ohky

File: docs/internals/contributing/new-contributors.txt [hunk 41]
```diff
@@ -148,7 +148,7 @@ some advice to make your work on Django more useful and rewarding.
   ensure that the *Needs tests*, *Needs documentation*, and *Patch needs
   improvement* flags are unchecked once you've addressed all review comments.
 
-  Remember that Django has an 8 month release cycle, so there's plenty of time
+  Remember that Django has an 8-month release cycle, so there's plenty of time
```
[the remark this answers is not in the document]
Comment (arslannoorpansota) [reply docs/internals/contributing/new-contributors.txt:151]: eight**??

File: docs/internals/contributing/writing-documentation.txt (hunk 35 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/writing-documentation.txt:362]: Ha, I read this proposed change backwards. I agree with your edit. I was confused and thought you were adding a comma. My mistake.

File: docs/internals/contributing/new-contributors.txt (hunk 41 above)
[the remark this answers is not in the document]
Comment (jacobtylerwalls) [reply docs/internals/contributing/new-contributors.txt:151]: I agree, `eight-month`

File: docs/internals/contributing/bugs-and-features.txt (hunk 31 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/bugs-and-features.txt:147]: I agree with Mariusz this is unnecessary to edit.

File: docs/internals/contributing/triaging-tickets.txt [hunk 42]
```diff
@@ -2,16 +2,16 @@
...
 pulled and sometimes there are flowers and vegetables that need picking. We need
-your help to sort out one from the other, and in the end we all benefit
+your help to sort out one from the other, and in the end, we all benefit
 together.
 
-Like all gardens, we can aspire to perfection but in reality there's no such
-thing. Even in the most pristine garden there are still snails and insects.
-In a community garden there are also helpful people who -- with the best of
+Like all gardens, we can aspire to perfection, but in reality there's no such
+thing. Even in the most pristine garden, there are still snails and insects.
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:13]:
```suggestion
thing. Even in the most pristine garden there are still snails and insects.
```

File: docs/internals/contributing/triaging-tickets.txt [hunk 43]
```diff
@@ -2,16 +2,16 @@
...
+In a community garden, there are also helpful people who -- with the best of
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:14]:
```suggestion
In a community garden there are also helpful people who -- with the best of
```

File: docs/internals/contributing/triaging-tickets.txt [hunk 44]
```diff
@@ -254,8 +254,8 @@ new contributions are made to the ticket.
 Keywords
 --------
 
-With this field you may label a ticket with multiple keywords. This can be
-useful, for example, to group several tickets of a same theme. Keywords can
+With this field, you may label a ticket with multiple keywords. This can be
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:257]:
```suggestion
With this field you may label a ticket with multiple keywords. This can be
```

File: docs/internals/contributing/triaging-tickets.txt [hunk 45]
```diff
@@ -414,8 +414,8 @@ Bisecting a regression
 
 .. highlight:: console
 
-A regression is a bug that's present in some newer version of Django but not in
-an older one. An extremely helpful piece of information is the commit that
+A regression is a bug that's present in some newer versions of Django but not
+in older ones. An extremely helpful piece of information is the commit that
```
Comment (jacobtylerwalls) [inline docs/internals/contributing/triaging-tickets.txt:418]: I don't see a benefit to changing this.

**Review by jacobtylerwalls — COMMENTED** [review]
Thanks for the improvements.

**Comment by jacobtylerwalls** [comment]
also maybe fix the typo in "grammar" in your title :D

File: docs/internals/contributing/committing-code.txt [hunk 46]
```diff
@@ -16,7 +16,7 @@ requests.
 
 When committing a pull request, make sure each individual commit matches the
 commit guidelines described below. Contributors are expected to provide the
-best pull requests possible. In practice however, committers - who will likely
+best pull requests possible. In practice, committers - who will likely
```
Comment (felixxm) [inline docs/internals/contributing/committing-code.txt:19]: I would leave `however`.

File: docs/internals/contributing/triaging-tickets.txt (hunk 45 above)
Comment (felixxm) [reply docs/internals/contributing/triaging-tickets.txt:418]: Me too.

Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:418]:
```suggestion
A Regression is a bug that's present in a newer version of Django but not
in older ones. An extremely helpful piece of information is the commit that
```

Comment (arslannoorpansota) [reply docs/internals/contributing/triaging-tickets.txt:418]:
```suggestion
A regression is a bug that's present in a newer version of Django but not
in older ones. An extremely helpful piece of information is the commit that
```

File: docs/internals/contributing/committing-code.txt (hunk 46 above)
Comment (arslannoorpansota) [reply docs/internals/contributing/committing-code.txt:19]:
```suggestion
best pull requests possible. In practice, however, committers - who will likely
```

File: docs/internals/contributing/triaging-tickets.txt (hunk 45 above)
Comment (jacobtylerwalls) [reply docs/internals/contributing/triaging-tickets.txt:418]: Still don't see much of a benefit, a regression can go unfixed for several versions, so "some newer versions" isn't wrong, and I would follow Mariusz's instincts here about minimizing work for translators and just fixing clear typos.

**Comment by arslannoorpansota** [comment]
@jacobtylerwalls Please have a look and proceed to merge!

File: docs/internals/contributing/committing-code.txt (hunk 39 above)
[the remark this answers is not in the document]
Comment (kezabelle) [reply docs/internals/contributing/committing-code.txt:19]: Perhaps I'm mis-reading @smithdc1's intent, but I think it was suggesting _removing_ the `however`, not to precede it with an additional comma. If I'm wrong, well, language is hard :)

**Comment by jacobtylerwalls** [comment]
Hi, it looks like some comments are still unaddressed.

https://github.com/django/django/pull/15195#discussion_r779629656
https://github.com/django/django/pull/15195#discussion_r774216886

**Review by jacobtylerwalls — APPROVED** [review]

**Comment by arslannoorpansota** [comment]
@jacobtylerwalls  It requires a maintainer to approve running workflows!

**Review by carltongibson — APPROVED** [review]
Thanks for this @arslannoorpansota. I rebased, and pushed small edits. 👍
